### ISSUE TITLE
[1.3.11rc2] Serial terminal has extra line breaks

### ISSUE THREAD

**Issue by kazibole** [foosel/OctoPrint#3115]
#### What were you doing?
Started a print from the printer's SD card via the printer's LCD panel.

#### What did you expect to happen?
Terminal shows communication without extra line breaks.

#### What happened instead?
Terminal shows communication with extra line breaks, random blank "Recv: " lines.

1.3.11rc1:

```
2019-04-08 17:36:54,780 - Connecting to: /dev/ttyACM0
2019-04-08 17:36:54,794 - Changing monitoring state from "Offline" to "Opening serial port"
2019-04-08 17:36:54,799 - Connected to: Serial<id=0x6a5775b0, open=True>(port='/dev/ttyACM0', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=20.0, xonxoff=False, rtscts=False, dsrdtr=False), starting monitor
2019-04-08 17:36:54,801 - Changing monitoring state from "Opening serial port" to "Connecting"
2019-04-08 17:36:54,821 - Send: N0 M110 N0*125
2019-04-08 17:36:54,823 - Recv: Command not found!
2019-04-08 17:36:55,918 - Recv: start
2019-04-08 17:36:55,932 - Send: N0 M110 N0*125
2019-04-08 17:36:55,939 - Recv: echo: 3.7.0-2201
2019-04-08 17:36:55,943 - Recv: echo: Last Updated: Apr  1 2019 20:56:14 | Author: (none, default config)
2019-04-08 17:36:55,947 - Recv: Compiled: Apr  1 2019
2019-04-08 17:36:55,951 - Recv: echo: Free Memory: 2037  PlannerBufferBytes: 1392
2019-04-08 17:36:55,955 - Recv: echo:Stored settings retrieved
2019-04-08 17:36:55,957 - Recv: adc_init
2019-04-08 17:36:56,295 - Recv: CrashDetect ENABLED!
2019-04-08 17:36:56,644 - Recv: PAT9125_RES_X=0
2019-04-08 17:36:56,647 - Recv: PAT9125_RES_Y=240
2019-04-08 17:36:56,650 - Recv: PAT9125_init:1
2019-04-08 17:36:56,653 - Recv: PAT9125_RES_X=0
2019-04-08 17:36:56,655 - Recv: PAT9125_RES_Y=240
2019-04-08 17:36:56,658 - Recv: PAT9125_init:1
2019-04-08 17:36:56,661 - Recv: FSensor ENABLED
2019-04-08 17:36:57,012 - Recv: echo:SD card ok
2019-04-08 17:36:57,549 - Recv: Error:No Line Number with checksum, Last Line: 0
2019-04-08 17:36:57,552 - Recv: Resend: 1
2019-04-08 17:36:57,560 - Recv: ok
```

1.3.12rc2:
```
2019-04-08 18:10:22,577 - Connecting to: /dev/ttyACM0
2019-04-08 18:10:22,675 - Changing monitoring state from "Offline" to "Opening serial port"
2019-04-08 18:10:22,718 - Connected to: Serial<id=0x6aa6fe70, open=True>(port='/dev/ttyACM0', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=20.0, xonxoff=False, rtscts=False, dsrdtr=False), starting monitor
2019-04-08 18:10:22,719 - Changing monitoring state from "Opening serial port" to "Connecting"
2019-04-08 18:10:22,746 - Send: N0 M110 N0*125
2019-04-08 18:10:34,950 - Recv: start

2019-04-08 18:10:34,965 - Send: N0 M110 N0*125
2019-04-08 18:10:34,970 - Recv: echo: 3.7.0-2201

2019-04-08 18:10:34,977 - Recv: echo: Last Updated: Apr  1 2019 20:56:14 | Author: (none, default config)

2019-04-08 18:10:34,980 - Recv: Compiled: Apr  1 2019

2019-04-08 18:10:34,983 - Recv: echo: Free Memory: 2037  PlannerBufferBytes: 1392

2019-04-08 18:10:34,986 - Recv: echo:Stored settings retrieved

2019-04-08 18:10:34,989 - Recv: adc_init

2019-04-08 18:10:35,326 - Recv: CrashDetect ENABLED!

2019-04-08 18:10:35,675 - Recv: PAT9125_RES_X=0

2019-04-08 18:10:35,679 - Recv: PAT9125_RES_Y=240

2019-04-08 18:10:35,682 - Recv: PAT9125_init:1

2019-04-08 18:10:35,685 - Recv: PAT9125_RES_X=0

2019-04-08 18:10:35,688 - Recv: PAT9125_RES_Y=240

2019-04-08 18:10:35,691 - Recv: PAT9125_init:1

2019-04-08 18:10:35,693 - Recv: FSensor ENABLED

2019-04-08 18:10:36,039 - Recv: echo:SD card ok

2019-04-08 18:10:36,575 - Recv: Error:No Line Number with checksum, Last Line: 0

2019-04-08 18:10:36,580 - Recv: Resend: 1

2019-04-08 18:10:36,588 - Recv: ok
```

#### Did the same happen when running OctoPrint in safe mode?
Yes.

#### Version of OctoPrint
Occurs in 1.3.11rc2
Did not occur in 1.3.11rc1

#### Operating System running OctoPrint
OctoPi 0.15.1
OctoPi 0.16.0

#### Printer model & used firmware incl. version
Prusa MK3 firmware 3.7.0

#### Browser and version of browser, operating system running browser
Chrome 73.0.3683.86 (64-bit)
Windows 10 1809

#### Link to octoprint.log
octoprint_1.3.11rc1.log https://gist.github.com/kazibole/f8e61b45cbeba44e10ab7b95c352557c
octoprint_1.3.11rc2.log https://gist.github.com/kazibole/055ef8ab07b0c575b9d9783008827bcb
octoprint_1.3.11rc2_safe.log https://gist.github.com/kazibole/ec267c6019694e9e89596e563b5e383a

#### Link to contents of terminal tab or serial.log
serial_1.3.11rc1.log https://gist.github.com/kazibole/b18f730713c044cb144ba7bef61fb24c
serial_1.3.11rc2.log https://gist.github.com/kazibole/0319511f8755888d1f94e719c1a75643
serial_1.3.11rc2_safe.log https://gist.github.com/kazibole/3fec066a4d1f63961a3f452168bb0ccb

#### Link to contents of Javascript console in the browser
Did not capture. Let me know if required.

#### Screenshot(s)/video(s) showing the problem:
Note: The screenshot is not from a print reflected in the logs attached.

![octoprint_blanks](https://user-images.githubusercontent.com/34734155/55771167-81034400-5a4c-11e9-853b-fed7c146b6a8.PNG)

I have read the FAQ.

**Comment by foosel** [comment]
Thanks for the thorough report! That was actually an easy fix and an obvious reason. Fixed by the above commit for 1.3.11rc3.

**Comment by foosel** [comment]
1.3.11rc3 is out which should fix this.
